### DISCUSSION TITLE
Add Setting on Receivers to change RSSI scale from dBm to PWM 'us'

### DISCUSSION THREAD

**Discussion by lfach** [ExpressLRS/ExpressLRS#2774] [Ideas]
For those who are using OLD flight controllers (like MFD Crosshair AP), it would be great to have receivers an option to translates the RSSI from 0 - -130dBm to PWM signal from 2000µs to 1000µs.

**Comment by pkendall64** [comment]
They do this already if you are not in 16ch mode.
Channel 15 is LQ scaled 0-100 into 1000-2000
Channel 16 is RSSIdBm scaled from -1xx (depending on packet rate) to -50 into 1000-2000.

**Comment by lfach** [comment]
Hi Paul,

Thanks for the reply.
Based on what you said, just to confirm:

- If I select SBUS on ERLS receiver, I can use ch15 to get RSSI/LQ in PWM
(us) scale from 1000-2000 ? Is that right?

Please confirm that, because there's a guy on Discord channel that told me
RSSI/LQ is not working for SBUS mode yet.

Thanks a lot.

Atenciosamente / Regards,

*Leandro Fachinelli*
_________________________________________


*Security and Infrastructure Specialist Consultant+55 19 99149-8822 *
***@***.**** ***@***.***>

**Comment by pkendall64** [comment]
Ah, not on SBUS, only on CRSF.

**Comment by lfach** [comment]
Is there any clue when this will be available on sbus?

**Comment by lfach** [comment]
Hello guys,

Any news about my request?

Atenciosamente / Regards,

*Leandro Fachinelli*
_________________________________________


*Security and Infrastructure Specialist Consultant+55 19 99149-8822 *
***@***.**** ***@***.***>

**Comment by XiannoFPV** [comment]
I'd love this feature as well. Have an old Pixfalcon running Arduplane and connected my elrs rx via SBUS and set it to output SBUS. Controls work like a charm but Arduplane needs the RSSI on CH16 as PWM.
It would be awesome if you could make it happen.
Thanks alot and best regards
Felix
